pcsx2/gui/MainFrame.cpp

```diff
@@ -637,35 +639,51 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
+	const CDVD_SourceType Source = g_Conf->CdvdSource;
+	const wxChar* SourceLabel[static_cast<uint8_t>(CDVD_SourceType::NumberOfSources)];
+	const wxChar* FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::NumberOfSources)];
+	const wxChar* BootIndicator[2][2];
+
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::Iso)] = _("Boot the VM using the current ISO source media");
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::Plugin)] = _("Boot the VM using the current CD/DVD source media");
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::NoDisc)] = _("Boot the VM without any source media");
+
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::Iso)] = _("ISO");
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::Plugin)] = _("CDVD");
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::NoDisc)] = _("BIOS");
+
+	BootIndicator[0][0] = _("Boot");
+	BootIndicator[0][1] = _("Reboot");
+	BootIndicator[1][0] = _(" (f&ull)");
+	BootIndicator[1][1] = _(" (&fast)");
+
+	wxString MenuItemLabel = static_cast<wxString>(BootIndicator[0][ActiveVM]) + " " + static_cast<wxString>(SourceLabel[static_cast<uint8_t>(Source)]);
```

Comment:
Some languages don't really use spaces (Chinese), so the `" "` shouldn't be placed here. Actually, breaking up a translatable string into multiple parts isn't a great idea - different languages have different word orders.

Also, those casts are unnecessary when you could just do `wxString(whatever)`.